src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1195,6 +1223,100 @@ def construct_payload(self):
         super().construct_payload()
         self.set_up_service_binds()
         self.set_up_extended_location()
+        self.set_up_source()
+        self.set_up_repo()
+
+    def set_up_source(self):
+        if self.get_argument_source():
+            self._set_up_source_or_repo()
+
+    def set_up_repo(self):
+        if self.get_argument_repo():
+            container_app = self._get_containerapp_if_exists()
```

Comment:
How about use a more meaningful name?
```suggestion
            existed_container_app = self._get_containerapp_if_exists()
```

Reply:
Updated